src/metrics/scenario.jl

```diff
@@ -60,19 +64,38 @@ end
 scenario_relative_cover = Metric(_scenario_relative_cover, (:timesteps, :scenarios))
 
 """
-    scenario_relative_juveniles(data::YAXArray, coral_spec::DataFrame, k_area::AbstractVector{<:Real}; kwargs...)::AbstractArray{<:Real}
+    scenario_relative_juveniles(data::YAXArray{<:Real,3}, coral_spec::DataFrame, k_area::AbstractVector{<:Real}; kwargs...)::AbstractArray{<:Real}
     scenario_relative_juveniles(rs::ResultSet; kwargs...)::YAXArray
 
 Calculate the mean relative juvenile population for each scenario for the entire domain.
+
+# Arguments
+- `X` : Raw data for a single scenario.
+- `coral_spec` : Coral spec DataFrame.
+- `k_area` : K_area.
+- `rs` : Resultset.
```

Comment:
I would put `rs` after `X` here, otherwise it gives the impression it comes after the other arguments.

By the way, these docstrings for `_metric_name` methods are generally temporary. They don't get picked up when querying `?metric_name` as the docstring is associated with the "wrong" method. There are ways to address this, but needs some investigation.

Reply:
Done